src/django/api/facility_type_processing_type.py

```diff
@@ -1,5 +1,10 @@
-from api.helpers import clean
-from thefuzz import process
+import warnings
+warnings.filterwarnings(
+    "ignore",
+    message="Using slow pure-python SequenceMatcher. Install python-Levenshte")
+
+from api.helpers import clean # NOQA
+from thefuzz import process # NOQA
```

Comment:
We can specify which lint rules specifically we want to opt out of, so we don't have to opt-out of lints we want to keep (e.g. "unused import")

```suggestion
from api.helpers import clean  # NOQA: E402
from thefuzz import process  # NOQA: E402
```